src/ProgressBar.jsx

```diff
@@ -1,18 +0,0 @@
-import React, {Component, PropTypes} from 'react'
```

Comment:
renamed this to Ticker because as I was implementing I realized it could own its state and just trigger the GameContainer on completion.